modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -71,32 +71,12 @@ def appellant_local_time
       signing_appellant.timezone ? created_at.in_time_zone(signing_appellant.timezone) : created_at.utc
     end
 
-    def veteran_first_name
-      auth_headers['X-VA-First-Name']
-    end
-
-    def veteran_middle_initial
-      auth_headers['X-VA-Middle-Initial']
-    end
-
-    def claimant_middle_initial
-      auth_headers['X-VA-Claimant-Middle-Initial']
-    end
-
-    def veteran_last_name
-      auth_headers['X-VA-Last-Name']
-    end
-
     def full_name
```

Comment:
rubocop complained once this became signing_appellant.first_name etc. because of line length